### ISSUE TITLE
Provide a root flowR script

### ISSUE THREAD

**Issue by EagleoutIce** [Code-Inspect/flowr#249]
We have many scripts (`slicer`, `benchmark`, `summarizer`, ...).
Instead of documenting all of them separately it would be nice to have a main root script named `flowR` which allows access to all of them.

**Comment by EagleoutIce** [comment]
This should be done before https://github.com/Code-Inspect/flowr/issues/166 so that the version flags etc can be part of that one root script.

**Comment by EagleoutIce** [comment]
It should be able to:

- [X] directly evaluate R expressions
- [ ] provide the current version of R
- [ ] allow to present just the ast or dataflow graph of a given input
- [ ] provide mermaid links if asked
